index.cs.md

```diff
@@ -0,0 +1,238 @@
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Více informací o Tvorbě přístupného webu" class="simple aside" %}
+{:/}
+
+-   Více o spolupracujících aspektech přístupnosti v [[Základních komponentech webové přístupnosti]](/fundamentals/components/).
+-   Pokyny pro zpřístupnění obsahu webu (WCAG), Pokyny pro zpřístupnění vývojových nástrojů (ATAG), ARIA Přístupné internetové aplikace a další důležité zdroje jsou představeny ve [[W3C Přehledu standardů přístupnosti]](/standards-guidelines/).
+-   Pro více informací o tom, jak W3C WAI vyvíjí materiály za pomoci několika zúčastněných stran, mezinárodní spolupráce a jak můžete přispět Vy, navštivte [[O WAI]](/about/) a [[Přispívání pro WAI]](/get-involved/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+### Zpřístupnění Vaší webové stránky {#website}
+
+Řada aspektů přístupnosti je poměrně snadno pochopitelná a impementovatelná. Některá řešení přístupnosti jsou složitější a jejich implementace vyžaduje více znalostí.
+
+Nejefektivnější je zařadit přístupnost hned od začátku projektu, aby nebylo nutné se vracet a opravovat již existující práci.
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Více informací o Zpřístupnění Vaší webové stránky" class="simple aside" %}
+{:/}
+
+-   Pro úvod do požadavků pro přístupnost a mezinárodních standardů, navštivte [[Principy přístupnosti]](/fundamentals/accessibility-principles/).
+-   Pro porozumění běžným překážkám přístupnosti z pohledu testování, navštivte [[Základní kontrola - První revize]](/test-evaluate/preliminary/).
+-   Pro základní činitele přístupnosti při navrhování, psaní a vyvíjení, navštivte [[Tipy pro začátek]](/tips/).
+-   Pokud jste připraveni dozvědět se víc o vyvíjení a navrhování, pravděpodobně se Vám budou hodit zdroje jako:
+    -   [Jak splnit WCAG (Rychlý průvodce)](http://www.w3.org/WAI/WCAG21/quickref/)
+    -   [Výukové materiály pro webovou přístupnost](https://www.w3.org/WAI/tutorials/)
+-   Pro řízení projektů a organizační aspekty, navštivte [[Plánování a řízení webové přístupnosti]](/planning-and-managing/).<br>
+    Pokud potřebujete rychlé opravy hned, navštivte [[Kroky pro dočasné opravy]](/planning/interim-repairs/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+	
+## Hodnocení přístupnosti {#evaluate}
+
+Při vývoji nebo redesignu webové stránky, zhodnoťte přístupnost na začátku, a pak také během vývoje, abyste identifikovali problémy v přístupnosti co nejdříve, dokud je možné je snadněji opravit. Jednuduché kroky, jako třeba změna nastavení v prohlížeči, Vám může pomoci zhodnotit některé aspekty přístupnosti. Komplexní zhodnocení toho, zda Vaše webová stránka splňuje všechny pokyny pro přístupnost, vyžaduje více úsilí.
```

Comment:
You can remove comma here: "Při vývoji nebo redesignu webové stránky, zhodnoťte přístupnost na začátku"

Reply:
Vám může pomoci -> Vám mohou pomoci